src/els_utils.erl

```diff
@@ -41,13 +41,14 @@ find_module(M, Extension) ->
 -spec find_document(uri()) ->
   {ok, els_document:document()} | {error, any()}.
 find_document(Uri) ->
-  case els_db:find(documents, Uri) of
-    {ok, Document} ->
+  case els_dt_documents:lookup(Uri) of
+    {ok, [#{document := Document}]} ->
       {ok, Document};
-    {error, not_found} ->
+    {ok, []} ->
+      %% TODO: Maybe we should not index this
```

Comment:
I am often getting timeouts when using this branch and navigating towards a OTP module during indexing.

Reply:
This was due to a bug in the db module. Fixed.